airflow/ui/src/pages/DagsList/DagsList.tsx

```diff
@@ -102,6 +103,17 @@ const columns: Array<ColumnDef<DAGResponse>> = [
     enableSorting: false,
     header: () => "Tags",
   },
+  {
+    accessorKey: "trigger",
+    cell: ({ row }) => (
+      <TriggerDAGModal
+        dagDisplayName={row.original.dag_display_name}
+        dagId={row.original.dag_id}
```

Comment:
As at a later point also the DAG params are needed - does it make sense to pass the full `row.original` object, not just the two strings?

Reply:
Yes, completely right, Will do all these changes once I add the functionality.

Reply:
In current scenario, we don't need more than dag_id,
Once we integrate API, At that time, if needed, we can add the parent.